Formula/infrakit.rb

```diff
@@ -0,0 +1,28 @@
+class Infrakit < Formula
+  desc "Toolkit for creating and managing declarative infrastructure"
+  homepage "https://github.com/docker/infrakit"
+  url "https://github.com/docker/infrakit.git",
+      :tag => "v0.5",
+      :revision => "3d2670e484176ce474d4b3d171994ceea7054c02"
+  head "https://github.com/docker/infrakit.git"
+
+  depends_on "pkg-config" => :build
+  depends_on "automake" => :build
+  depends_on "libvirt" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/docker/infrakit").install buildpath.children
+    cd "src/github.com/docker/infrakit" do
+      system "make", "get-tools"
```

Comment:
This is running `go get` which isn't allowed in core.

Reply:
Ok any suggestions to get the dependencies? The other way to build the binary is within a docker container but I suspect that is disallowed too. `wget -qO- https://docker.github.io/infrakit/install | sh`